tests/simtel/test_simtelfile.py

```diff
@@ -128,21 +128,22 @@ def test_iterate_complete_file_zst():
 
 def test_iterate_mc_events():
     expected = 200
-    with SimTelFile(prod4_path) as f:
-        for counter, event in enumerate(f.iter_mc_events(), start=1):
+    with SimTelFile(prod4_path, skip_non_triggered=False) as f:
```

Comment:
I just realize, my brain cannot parse this double negative `skip_non_triggered=False`. I think the inverse `include_non_triggered=True` would be easier to read.

Reply:
The we have the double negative inside in the `skip`. But I assume better inside than in the API.

Reply:
Well ... maybe it is only me ... I just needed to do a double take and think hard :D ..

Reply:
You mean this:
```
 def _check_skip(self, event):
        if event['type'] == 'data':
            return self.skip_non_triggered and not event.get('telescope_events')
```
would change to
```
 def _check_skip(self, event):
        if event['type'] == 'data':
            return not self.include_non_triggered and not event.get('telescope_events')
```
or to 
```
 def _check_skip(self, event):
        if event['type'] == 'data':
            return not (self.include_non_triggered or event.get('telescope_events'))
```
right?

I honestly think all 3 versions look difficult :D.

Reply:
I made a small PR #215 into this branch with an alternative proposal .. maybe it is better. ... no idea.
please check and feel free to reject and delete PR #215

Reply:
Discussed in #215.

We keep the `skip_xxx` interface, since the users are already used to it and we do not want to break API too much here.